### PR TITLE
[LD-622][Malika] Added sentence casing for api doc

### PR DESCRIPTION
**Pull request by malikaansari91**
Ticket: https://apollopde.atlassian.net/browse/LD-622

#### What does this PR do?
Added Sentence casing for labels and text

Make sure you've checked off all these things before submitting:

- [ ] This pull request isn't for a company's fork, it's intended for the upstream Slate shared by everybody.
- [x] This pull request is submitted to the `master` branch.
- [ ] If it makes frontend changes, this pull request has been tested in the latest version of Firefox, Chrome, IE, and Safari.

image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

File: source/includes/_contacts.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ contact_stage_id | Assign contact to this stage, if the contact does not yet exi
 present_raw_address | The address string for this contact, Apollo will intelligently infer the city, state, country, and time zone from your address | "San Francisco"
 
 
-## Update a Contact
+## Update a contact
```
Comment (MauleePhadke19) [inline source/includes/_contacts.md:132]: Casing update the description section on line no [121](https://github.com/apolloio/apollo-api-docs/blob/80a972fb8380367506d3f84e6096153eca28fd2c/source/includes/_contacts.md?plain=1#L121) and [122](https://github.com/apolloio/apollo-api-docs/blob/80a972fb8380367506d3f84e6096153eca28fd2c/source/includes/_contacts.md?plain=1#L122) to be consistent with [120](https://github.com/apolloio/apollo-api-docs/blob/80a972fb8380367506d3f84e6096153eca28fd2c/source/includes/_contacts.md?plain=1#L120)

`Last Name` -> `Last name`
`Company Name` -> `Company name`

File: source/includes/_contacts.md [hunk 2]
```diff
@@ -265,7 +265,7 @@ home_phone | The home phone for this contact. If the contact already has a home
 other_phone | Phone of unknown type for this contact. If the contact already has phone of unknown type, this overwrites it. If a contact does not already have phone of unknown type, this ADDS a new phone on top of existing phone numbers | "123-456-7890"
 
 
-## Search for Contacts
+## Search for contacts
```
Comment (MauleePhadke19) [inline source/includes/_contacts.md:268]: Casing update the description section on line no [254](https://github.com/apolloio/apollo-api-docs/blob/80a972fb8380367506d3f84e6096153eca28fd2c/source/includes/_contacts.md?plain=1#L254) and [255](https://github.com/apolloio/apollo-api-docs/blob/80a972fb8380367506d3f84e6096153eca28fd2c/source/includes/_contacts.md?plain=1#L255) to be consistent with [253](https://github.com/apolloio/apollo-api-docs/blob/80a972fb8380367506d3f84e6096153eca28fd2c/source/includes/_contacts.md?plain=1#L253)

`Last Name` -> `Last name`
`Company Name` -> `Company name`

File: source/includes/_contacts.md [hunk 3]
```diff
@@ -460,7 +460,7 @@ sort_by_field | Possible values: "contact_last_activity_date", "contact_email_la
 sort_ascending | Possible values: true or false
 page | which page to return. Defaults to 1
```
Comment (MauleePhadke19) [inline source/includes/_contacts.md:461]: Sentence case for description: First letter should be capital
`which page to return` -> `Which page to return`

File: source/includes/_opportunities.md [hunk 4]
```diff
@@ -81,7 +81,7 @@ print(response.text)
 | closed_date          | The date the Opportunity was closed                                                                                                         |
```
Comment (MauleePhadke19) [inline source/includes/_opportunities.md:81]: Line no 78 
`Opportunity Name` -> `Opportunity name` 

Line no 79
`The Amount of money involved in the Opportunity/ Deal` -> `The amount of money involved in the opportunity/ deal`

Line no 80 
`The ID of the current Stage .You can get a list of all opportunity stages from Misc/Opportunity Stages` -> `The ID of the current stage .You can get a list of all opportunity stages from misc/opportunity stages` 

Line no 81
`The date the Opportunity was closed ` -> `The date the opportunity was closed `

Line no 82
`ID of the Account ` -> `ID of the account`

File: source/includes/_opportunities.md [hunk 5]
```diff
@@ -232,7 +232,7 @@ This endpoint returns all opportunities you have Created.
 
 `GET https://api.apollo.io/v1/opportunities`
 
-## View Opportunity
+## View opportunity
```
Comment (MauleePhadke19) [inline source/includes/_opportunities.md:235]: Line no 231 
`This endpoint returns all opportunities you have Created.` -> `This endpoint returns all opportunities you have created.`

File: source/includes/_opportunities.md [hunk 6]
```diff
@@ -295,7 +295,7 @@ You can GET a list of all created Opportunities from [Opportunities/all](#get-al
 
 `GET https://api.apollo.io/v1/opportunities/REPLACE_WITH_OPPORTUNITY_ID`
 
-## Update Opportunity
+## Update opportunity
```
Comment (MauleePhadke19) [inline source/includes/_opportunities.md:298]: Line no 294 
`You can GET a list of all created Opportunities from Opportunities/all` -> `You can GET a list of all created opportunities from opportunities/all`

Comment (MauleePhadke19) [reply source/includes/_opportunities.md:298]: Line no 365
Same as above

File: source/includes/_people.md [hunk 7]
```diff
@@ -252,14 +252,14 @@ This endpoint searches for people and do not return any email information. To ge
...
 `POST https://api.apollo.io/v1/mixed_people/search`
 
-### Query Parameters
+### Query parameters
 
 Parameter | Description | Example
 --------- | ----------- | -----------
 person_titles (optional) | an array of the person's title. Apollo will return results matching ANY of the titles passed in |  ["sales director", "engineer manager"]
 q_organization_domains (optional) | an array of the company domains to search for, joined by the new line character.  | "google.com\nfacebook.com"
 page (optional) | an integer that allows you to paginate through the results  | 1
```
Comment (MauleePhadke19) [inline source/includes/_people.md:261]: Capitalize the first letter for all description items. It is consistent with other pages

File: source/includes/_ratelimits.md [hunk 8]
```diff
@@ -24,9 +24,9 @@ To view how many requests your current plan allows, as well as how many requests
 We use a fixed-window rate limiting strategy for calls to our API. If your team's rate limit is 50 requests per minute, those 50 api calls can be made at any interval within the 60 second window.
 </aside>
 
-### Response Headers
+### Response headers
 
-Header Name | Description | Example
+Header name | Description | Example
 ----------- | ----------- | -----------
 x-daily-requests-left | the total requests left for the current 24 hour period |  295
```
Comment (MauleePhadke19) [inline source/includes/_ratelimits.md:31]: Let's capitalize the first letter for all description items. It should be consistent with other pages

**Review by MauleePhadke19 — APPROVED** [review]
LGTM

**Review by abhishekApollo — APPROVED** [review]
LGTM !

**Pull request merged**